samples/src/main/scala/SparkStreamingKustoSink.scala

```diff
@@ -19,40 +21,56 @@ object SparkStreamingKustoSink {
       .getOrCreate()
 
     // read messages from Azure Event Hub
-    val connectionString = ConnectionStringBuilder("Event Hub Connection String")
-      .setEventHubName("Event Hub Name")
+    val connectionString = ConnectionStringBuilder("Endpoint=sb://bittonhub.servicebus.windows.net/;SharedAccessKeyName=KustoReadKey;SharedAccessKey=PSPCQgAtQGS02sQ77PxT+lJDBRvcnrlx4IZJP+vSep0=")
+      .setEventHubName("ehbitton")
       .build
 
     val eventHubsConf = EventHubsConf(connectionString)
-      .setStartingPosition(EventPosition.fromEndOfStream)
+      .setStartingPosition(EventPosition.fromEndOfStream).setMaxEventsPerTrigger(5)
```

Comment:
Looks like some inwork experimentation... if not - please explain F2F, and add comments